### PR TITLE
add fetchCharacters function

### PR DESCRIPTION
**Pull request by ninaassmann**

### CODE REVIEW

**Comment by chris-kuehn** [comment]
perfect! :)

**Comment by jonas8900** [comment]
perfect!

**Review by KilianBau — COMMENTED** [review]
Looks good 💯

File: index.js [hunk 1]
```diff
@@ -12,3 +18,38 @@ const pagination = document.querySelector('[data-js="pagination"]');
...
+    const response = await fetch(url);
+    console.log(response);
+    if (response.ok) {
+      const result = await response.json();
+      console.log(result);
+
+      const characters = result.results;
+      console.log(characters);
+
+      characters.forEach((character) => {
```
Comment (jonas8900) [inline index.js:38]: good idea!

**Review by jonas8900 — COMMENTED** [review]
lgtm!

**Comment by jonas8900** [comment]
test

**Pull request merged**